pkg/datamover/datamover.go

```diff
@@ -136,7 +136,7 @@ func DeleteTempVSClass(backupName string, tempVS snapshotv1listers.VolumeSnapsho
 
 	tempVSClass, err := tempVS.Get(fmt.Sprintf("%s-snapclass", backupName))
 	if err != nil {
-		log.Errorf("failed to get temp vsClass %v", tempVSClass.Name)
```

Comment:
Maybe you frame the tempVSClass Name first in a local var, that way you can use that var in the log to specify the vsclass name ?

Reply:
The issue is this temp VSClass does not get created if VSB fails early, so assigning a var to this name would be nil in this case, which then throws an error.

Reply:
So what I meant was use,
`tempVSClassName := fmt.Sprintf("%s-snapclass", backupName)` then use `tempVSClassName` in the logs. We should be having the backupName and `tempVSClassName` is generated per backup.